examples/authorized_api/authorization.go

```diff
@@ -18,6 +67,54 @@ func MustAuthorize(handler http.HandlerFunc) http.HandlerFunc {
 	}
 }
 
+func ZebedeeMustAuthorize(handler http.HandlerFunc) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		status, err := zebedeeValidateAuth()
+		if err != nil {
+			if status == "401 Unauthorized" {
+				http.Error(w, err.Error(), http.StatusUnauthorized)
+			} else {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
+			}
+			return
+		} else {
+			w.WriteHeader(200)
+			message := "[\"DELETE\", \"READ\", \"CREATE\", \"UPDATE\"]"
+			fmt.Fprintf(w, message)
+		}
```

Comment:
Else not necessary here see previous comment.

Reply:
done